docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
 	| Vertiq 2306 | .. centered:: |:x:|                |
 	+-------------+------------------------------------+
 
+.. _standard_dronecan_support:
+
 Standard DroneCAN Support
 ================================
-Vertiq modules that support DroneCAN support a standard set of messages, requests, and configuration parameters. For information on the DroneCAN messages, 
-requests, and parameters supported by Vertiq modules, see the `Standard DroneCAN Support documentation <https://www.vertiq.co/s/Preliminary-Standard-DroneCAN-UAVCANv0-Support.pdf>`_ 
-on Vertiq’s website.
+This section details the standard DroneCAN messages supported across all Vertiq DroneCAN modules currently. The structure and contents of these messages are defined by the 
+DroneCAN specification, this section just provides details on how exactly Vertiq modules support them and some supplementary information. 
+
+Details on the DroneCAN protocol can be found on the `DroneCAN specification <https://dronecan.github.io/Specification/1._Introduction/>`_. For a full listing of all standard messages 
+specified by DroneCAN, see the `List of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/>`_ in the DroneCAN specification.
+
+Broadcast Messages
+********************
+These are broadcast messages that are sent to or from the motors. Since they are broadcast, there is no response message sent. These messages typically make up the majority of 

```

Comment:
I'd define what broadcast means. I know people who are using dronecan on purpose probably know, but it might be a good addition for people reading through what it can do.